Specification/BrAPI-Schema/BrAPI-Core/Study.json

```diff
@@ -66,90 +74,38 @@
-                            "value": {
-                                "description": "Numerical or categorical value\n\nMIAPPE V1.1 (DM-59) Environment parameter value - Value of the environment parameter (defined above) constant within the experiment.",
-                                "type": [
-                                    "null",
-                                    "string"
-                                ]
-                            },
-                            "valuePUI": {
-                                "description": "URI pointing to an ontology class for the parameter value",
-                                "type": [
-                                    "null",
-                                    "string"
-                                ]
-                            }
-                        },
-                        "required": [
-                            "parameterName",
-                            "description"
-                        ],
-                        "type": "object"
+                        "$ref": "../BrAPI-Core/Study.json#/$defs/EnvironmentParameters",
+                        "description": "EnvironmentParameters"
                     },
+                    "title": "EnvironmentParameters",
                     "type": [
                         "null",
                         "array"
                     ]
                 },
                 "experimentalDesign": {
                     "description": "The experimental and statistical design full description plus a category PUI taken from crop research ontology or agronomy ontology",
-                    "properties": {
-                        "PUI": {
-                            "description": "MIAPPE V1.1 (DM-23) Type of experimental design - Type of experimental  design of the study, in the form of an accession number from the Crop Ontology.",
-                            "type": [
-                                "null",
-                                "string"
-                            ]
-                        },
-                        "description": {
-                            "description": "MIAPPE V1.1 (DM-22) Description of the experimental design - Short description of the experimental design, possibly including statistical design. In specific cases, e.g. legacy datasets or data computed from several studies, the experimental design can be \"unknown\"/\"NA\", \"aggregated/reduced data\", or simply 'none'.",
-                            "type": [
-                                "null",
-                                "string"
-                            ]
-                        }
+                    "relationshipType": "one-to-many",
+                    "referencedAttribute": "study",
+                    "items": {
+                        "$ref": "../BrAPI-Core/Study.json#/$defs/ExperimentalDesign",
+                        "description": "ExperimentalDesign"
                     },
```

Comment:
Model change from a single ExperimentalDesign to an array of ExperimentalDesign. Not sure this is appropriate.